### ISSUE TITLE
buy function reversed token name?

### ISSUE THREAD

**Issue by okwme** [gnosis/dx-services#51]
I'm trying to buy RDN for WETH so i ran:
```
yarn cli-rinkeby buy 5 RDN-WETH
```
and received the message:
```
yarn run v1.13.0
$ NETWORK=rinkeby node src/cli/cli.js buy 5 RDN-WETH
Keccak bindings are not compiled. Pure JS implementation will be used.
nonce tracker getWeb3.js
  INFO-dx-service:repositories:EthereumClient Checking Node connectivity... +0ms

  INFO-dx-service:repositories:EthereumClient Using Node Geth/v1.8.20-stable/linux-amd64/go1.11.2 +1s
  INFO-cli Buy 5 WETH on RDN-WETH (auction 73) using the account 0xee4e56947c799127fb37392bf9333bdef356865f +0ms
```
I canceled the transaction because I assumed I got it backwards. I did not want to "Buy 5 WETH", I wanted RDN. So then I ran the opposite:
```
yarn cli-rinkeby buy 5 WETH-RDN
```
and received the message:
```
yarn run v1.13.0
$ NETWORK=rinkeby node src/cli/cli.js buy 5 WETH-RDN
Keccak bindings are not compiled. Pure JS implementation will be used.
nonce tracker getWeb3.js
  INFO-dx-service:repositories:EthereumClient Checking Node connectivity... +0ms

  INFO-dx-service:repositories:EthereumClient Using Node Geth/v1.8.20-stable/linux-amd64/go1.11.2 +920ms
  INFO-cli Buy 5 RDN on WETH-RDN (auction 73) using the account 0xee4e56947c799127fb37392bf9333bdef356865f +0ms
cli.js buy <amount> <token-pair>

Buy in a auction for a token pair

Positionals:
  amount      Amount to buy, i.e. 0.8                                                     [required]
  token-pair  The token pair of the auction, i.e. WETH-RDN                       [string] [required]

Options:
  --version  Show version number                                                           [boolean]
  -h         Show help                                                                     [boolean]

{ AssertionError [ERR_ASSERTION]: The user 0xee4e56947c799127fb37392bf9333bdef356865f has just 0 RDN (required 5 RDN)
    at AuctionRepoImpl._assertBalanceAux (/Users/billy/gnosis/dx-services/src/repositories/AuctionRepo/AuctionRepoImpl.js:1052:5)
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
  generatedMessage: false,
  name: 'AssertionError [ERR_ASSERTION]',
  code: 'ERR_ASSERTION',
  actual: false,
  expected: true,
  operator: '==' }
error Command failed with exit code 1.
```
Of course I have 0 RDN, that's why I want to buy it.

Not sure if the terminology of the command should be changed, or the result of the action should be changed tho. generally token pairs go `sellerToken-buyerToken` right? so intuitively that makese sense to me cause I'd want to write "Buy sellerToken (sell buyerToken)". on the flip side it also make sense because when I sell I'd want to "Sell sellerToken (buy buyerToken)".

**Comment by dasanra** [comment]
Yes it's always `sellerToken-buyerToken` we should review if maybe the log message was not set correctly and can lead to confusion.

Thanks for report

**Comment by dasanra** [comment]
We rewrote the message trying to make more clear what you are buying and what are you spending.
 Is available now in `staging` version and will be available in stable versions from `2.1`

https://github.com/gnosis/dx-services/commit/6354cabaff0825e53a947095a02ac6a6e2d2e35b
